onnxscript/ir/__init__.py

```diff
@@ -69,11 +69,12 @@
     # IR Tensor initializer
     "tensor",
     # Pass infrastructure
+    "_external_data",
```

Comment:
A private module should not be exposed here. We can have a follow up to decide how we want to expose this